sdk/core/core-amqp/package.json

```diff
@@ -53,9 +53,9 @@
     "format": "dev-tool run vendored prettier --write --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
     "integration-test": "npm run integration-test:node && npm run integration-test:browser",
     "integration-test:browser": "echo skipped",
-    "integration-test:node": "echo skipped",
-    "lint": "eslint package.json api-extractor.json src test",
-    "lint:fix": "eslint package.json api-extractor.json src test --fix --fix-type [problem,suggestion]",
+    "integration-test:node": "dev-tool run test:vitest --no-test-proxy --esm",
```

Comment:
I would be happier if we had instead of integration tests, you can introduce type tests instead, such as `test:type` or `unit-test:type`

Reply:
I skipped them for now in https://github.com/Azure/azure-sdk-for-js/pull/32099/commits/56e514b289ff5b3482ce1ffe4666e32a03d1f203.